src/windows/common/tools/README.md

```diff
@@ -0,0 +1,14 @@
+#### Tools included in this folder:
+
+File Name: kbdgctrl.exe
```

Comment:
![image](https://user-images.githubusercontent.com/6799284/115287433-1db14f80-a105-11eb-9c48-bc127442f445.png)

can you use a markdown editor or use git hub to preview the markdown to format these to be more easily readable?